transmart-rest-api/build.gradle

```diff
@@ -29,8 +29,8 @@ dependencies {
     testCompile "org.grails:grails-plugin-testing"
     testRuntime "net.sourceforge.htmlunit:htmlunit:2.18"
     testCompile 'org.grails:grails-datastore-rest-client'
-    compile 'org.quartz-scheduler:quartz:2.2.1'
     compile "com.opencsv:opencsv:${openCsvVersion}"
+    compile "org.grails.plugins:quartz:${quartzVersion}"
```

Comment:
Why you introduced this dependency for?

Reply:
Because it is required when running transmart-server without transmartApp. Not really related to the JSON writer, but I encountered this issue during testing (I was running without transmartApp for having a more light weight application).